src/containers/Bounty/components/modals/FulfillerApplicationFormModal.js

```diff
@@ -64,7 +64,7 @@ let FulfillerApplicationModalComponent = props => {
             </Text>
           )}
           {status === 'api error' && (
-            <Text inputLabel lassName={styles.modalSubmitError}>
+            <Text inputLabel className={styles.modalSubmitError}>
```

Comment:
Need to add linter back in.. even if we don't block via git commit hook initially